neo/Persistence/LevelDB/Prefixes.cs

```diff
@@ -19,5 +19,6 @@ internal static class Prefixes
         public const byte IX_CurrentHeader = 0xc1;
 
         public const byte SYS_Version = 0xf0;
+        public const byte CN_Context = 0xf4;
```

Comment:
It is not appropriate to put consensus-related constants here.

Reply:
I had it here since that is where all the storage Prefixes are. Should we make another file for the consensus related ones?

Reply:
Maybe Helper.cs ?

Reply:
If a separate storage context is not given for consensus, the prefix for consensus context is in the same key space as the other `Prefixes`. For this reason it is odd to not have it here in the same place where someone would go to see what prefixes are already in use.

Reply:
We can add a comment to `Prefixes` to indicate that a range of prefixes is reserved for external use.

Reply:
That sounds good. What range do you suggest we specify is reserved for external use.

Reply:
`0xe0~0xef`?

Reply:
`SYS_VERSION` is using `0xf0`; how about `0xf1-0xff`?

Also, why are we considering the Consensus external use, when it is part of the core code?

Reply:
`neo/Persistence/LevelDB/Prefixes.cs` This file is specific to leveldb storage.

`0xf1-0xff` is okay.

Reply:
I still think it is strange to consider the prefix for the consensus state external use, when the consensus mechanism is required for the network to operate; however, I went ahead and moved the constant for it to Helper.cs (as you suggested) and left a comment in `neo/Persistence/LevelDB/Prefixes.cs`.